server/src/interfaces/media.interface.ts

```diff
@@ -32,6 +32,7 @@ interface DecodeImageOptions {
 export interface DecodeToBufferOptions extends DecodeImageOptions {
   size: number;
   orientation?: ExifOrientation;
+  isEmbeddedImage?: boolean;
```

Comment:
I was thinking of just changing `const orientation = Number(asset.exifInfo?.orientation) || undefined;` to `const orientation = useExtracted && asset.exifInfo?.orientation ? Number(asset.exifInfo.orientation) : undefined;` I don't think you need anything else.

Reply:
ah, this is cleaner